### ISSUE TITLE
Loss is nan？

### ISSUE THREAD

**Issue by hit-dyl** [PRIS-CV/Fine-Grained-or-Not#8]
hello,why does the training loss keep increasing and gradually become nan？
(python 3.8  torch 1.7 torchvision 0.8)

**Comment by dongliangchang** [comment]
Please refer to this issue for details #2
